src/modals/AddTaskModal.js

```diff
@@ -35,15 +36,25 @@ export default class AddTaskModal extends Component<{}> {
 
   constructor(props){
     super(props);
+    this.formatSelectedDate = this.formatSelectedDate.bind(this);
     this.state = {
       taskText: '',
-      date: currentDate
+      date: currentDate,
+      milliDate: ''
     }
   }
 
+  formatSelectedDate(date){
+    const dateChangeMilli = new Date( Date.parse(date) );
+    let month = monthDate[dateChangeMilli.getMonth()];
+    let dateChangeFormat = month + ' ' + dateChangeMilli.getDate();
+    this.state.milliDate = dateChangeFormat;
```

Comment:
Nunca nunca nunca debemos alterar directamente el estado de un componente de esta forma, para eso tenemos this.setState() y aún así no entiendo para que guardar esta nueva propiedad en el estado...